doc/manpages/man5/afp.conf.5.md

```diff
@@ -531,6 +531,69 @@ vol dbnest = *BOOLEAN* (default: *no*) **(G)**
 the CNID database in a folder called .AppleDB inside the volume root of
 each share.
 
+## Directory Cache Tuning
```

Comment:
Random thought: This topic may even deserve a section in the Configuration manual chapter. What do you think?

Reply:
Ha. Sure.. Where is the Configuration manual chapter?
I'm going to squash all these commits and do a final code quality check anyway. I can move it over then